### PR TITLE
Pfm 1134 - Attendence View Screen

### PR DESCRIPTION
**Pull request by hariomsinha-eGov**

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/AttendenceMgmt/src/pageComponents/AttendanceDateRange.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  const handleFocusChange = (focusedRange) => {
+    const [rangeIndex, rangeStep] = focusedRange;
+    setFocusedRange(focusedRange);
+  };
+
+  const handleClose = () => {
+    setIsModalOpen(false);
+  };
+
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/AttendenceMgmt/src/pageComponents/AttendanceDateRange.js:153]: You can simply import this component from react-comp library if no customization is required @hariomsinha-eGov

Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/AttendenceMgmt/src/pageComponents/AttendanceDateRange.js:153]: As discussed, the new file was added due to some underlying changes required based on the UX.

**Review by nipunarora-eGov — APPROVED** [review]

**Comment by jagankumar-egov** [comment]
also has some merge conflicts

File: frontend/micro-ui/web/micro-ui-internals/example/src/index.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ import { initLibraries } from "@egovernments/digit-ui-libraries";
 import { initDSSComponents } from "@egovernments/digit-ui-module-dss";
 import { initEngagementComponents } from "@egovernments/digit-ui-module-engagement";
 import { initWorksComponents } from "@egovernments/digit-ui-module-works";
-import { initAttendenceMgmtComponents } from "@egovernments/digit-ui-module-attendencemgmt";
+import { AttendenceMgmtModule, initAttendenceMgmtComponents } from "@egovernments/digit-ui-module-attendencemgmt";
```
[the remark this answers is not in the document]
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/example/src/index.js:11]: Yes, I have removed it.

File: frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Header.js [hunk 3]
```diff
@@ -1,7 +1,12 @@
 import React from "react";
 
 const Header = (props) => {
-  return <header className="h1" style={props.styles}>{props.children}</header>;
+  console.log(props.children);
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Header.js:4]: remove console logs even commented one

Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Header.js]: done

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**